pkgs/development/libraries/rapidyaml/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, cmake
+, fetchFromGitHub
+, git
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rapidyaml";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "biojppm";
+    repo = pname;
+    fetchSubmodules = true;
+    rev = "v${version}";
+    sha256 = "sha256-1/P6Szgng94UU8cPFAtOKMS+EmiwfW/IJl2UTolDU5s=";
+  };
+
+  nativeBuildInputs = [ cmake git ];
```

Comment:
Can we avoid using git?

Reply:
It's used here: https://github.com/biojppm/cmake/blob/52a4f4e64993589673b35b78b52b9212cb848d3a/c4Project.cmake#L1142

Reply:
Ideally we want to hardcode those values and not rely on git as the git information should normally not be present as it isn't that highly reproducible as the plain source download.

Reply:
Git is required for a function of a submodule (that is in turn a submodule).
It's not actually used in the main project that we care about.